### DISCUSSION TITLE
Lazy loading

### DISCUSSION THREAD

**Discussion by chinespec101** [mdbootstrap/TW-Elements#1518] [Share ideas & request features]
I would like to have an easy to implement delayed loading for images that are not immediately visible in the viewport.

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1563

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
Good news! We introduced this feature in our latest release!
See here: #1899
